src/lib/icon/icon.ts

```diff
@@ -202,4 +212,13 @@ export class MatIcon extends _MatIconMixinBase implements OnChanges, OnInit, Can
       this._previousFontIconClass = this.fontIcon;
     }
   }
+
+  /**
+   * Cleans up a value to be used as a fontIcon or fontSet.
+   * Since the value ends up being assigned as a CSS class, we
+   * have to trim the value and omit space-separated values.
+   */
+  private _cleanupFontValue(value: string) {
+    return typeof value === 'string' ? value.trim().split(' ')[0] : value;
```

Comment:
Looks like ligatures can include whitespace:
https://twitter.com/larrythefox/status/422813021807714304

Reply:
Our current logic still doesn't hold up though. We used to take the value verbatim and set it as a class which is invalid. I switched it up to only take it up to the first space.

Reply:
Oh, right. I mixed up class and ligature